core/domain/prod_validation_jobs_one_off_test.py

```diff
@@ -424,6 +430,247 @@ def test_model_with_invalid_id(self):
         run_job_and_check_output(self, expected_output, sort=True)
 
 
+class ClassifierTrainingJobModelValidatorTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ClassifierTrainingJobModelValidatorTests, self).setUp()
+
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+
+        explorations = [exp_domain.Exploration.create_default_exploration(
+            '%s' % i,
+            title='title %d' % i,
+            category='category%d' % i,
+        ) for i in xrange(2)]
+
+        for exp in explorations:
+            exp.add_states(['StateTest%s' % exp.id])
+            exp_services.save_new_exploration(self.owner_id, exp)
+
+        next_scheduled_check_time = datetime.datetime.utcnow()
+        id0 = classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', '0', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'StateTest0', feconf.TRAINING_JOB_STATUS_NEW,
+            None, 1)
+        self.model_instance_0 = (
+            classifier_models.ClassifierTrainingJobModel.get_by_id(id0))
+        id1 = classifier_models.ClassifierTrainingJobModel.create(
+            'CodeClassifier', 'CodeRepl', '1', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'StateTest1', feconf.TRAINING_JOB_STATUS_NEW,
+            None, 1)
+        self.model_instance_1 = (
+            classifier_models.ClassifierTrainingJobModel.get_by_id(id1))
+
+        self.job_class = (
+            prod_validation_jobs_one_off.ClassifierTrainingJobModelAuditOneOffJob) # pylint: disable=line-too-long
```

Comment:
As with elsewhere, try to remove unnecessary pragmas here and below.

Reply:
Done.